docker-container/contents/container-inspect

```diff
@@ -2,6 +2,7 @@
 
 import argparse
 import logging
+import shlex
```

Comment:
shlex.split() for splitting strings is better than plain split() because it will not split spaces that are wrapped in quotes, similar to shell. 

This can be seen in an example like below...

```python
# Good
shlex.split('--env=MYVAR="My var" --env=OTHERVAR=OTHERVAR')

[
    '--env=MYVAR="My var"', 
    '--env=OTHERVAR=OTHERVAR'
]

# Bad
'--env=MYVAR="My var" --env=OTHERVAR=OTHERVAR'.split()

[
    '--env=MYVAR="My', 
    'var"', 
    '--env=OTHERVAR=OTHERVAR'
]
```